src/contracts.ts

```diff
@@ -362,35 +374,34 @@ export async function fetchMirrorFunds(ctx: Context) {
  * @param {optOutOfAirdrop} params - optout parameters
  */
 export async function optOutOfAirdrop(params: OptOutOfAirdropInterface) {
+  console.log('Opting you out of airdrop')
+  const { cAddress, network, wallet, derivationPath, pvtKey, transactionId } = params
+  if (await isOptedOut(cAddress, network)) {
+    throw new Error('You have already opted out of Airdrop')
```

Comment:
I will exit the cli loop in interactive mode. Can you just log the msg and return from function?